### DISCUSSION TITLE
GEFSv13?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/global-workflow#2894] [Q&A] [answered]
Is it possible to run GEVSv13 via the g-w? I didn't see it as a tag or branch - should I just run from head of develop at this point? 

[updated for clarity]

**Comment by aerorahul** [comment, marked as answer]
GEFSv13 science configuration is still in flux.  One can run an ensemble forecast under a GEFS configuration from `develop`.  
Tests are being added to run from replay as well.

**Reply by benjamin-cash** [reply]
Thanks!
